css/services.css

```diff
@@ -0,0 +1,11 @@
+.services__list {
+  display: flex;
+  gap: 30px;
+
+  list-style: none;
```

Comment:
Reset it in `reset.css` and you won't need to repeat this line of code for every `ul` in the project